### ISSUE TITLE
Link to category from an app page

### ISSUE THREAD

**Issue by nclm** [flathub-infra/website#4268]
**Use case:**

I was visiting https://flathub.org/apps/net.hhoney.rota and wanted to see more games. I was surprised not to find any way to do that from the page. There are tags underneath, but they are not helpful in that case. The only way I found was to go the the homepage, and scroll until I find games, and click on More Games.

**Suggestion:**

Add a link to the category from an app page. It can take many possible forms and can be located in many possible places, but the fact that an app page displays no category or link to it seems like something that got forgotten?

**Comment by razzeee** [comment]
Categories are not very good in general, as they haven't been updated in decades - if anything the plan is to remove/replace them completely. Way too many apps are `utility` for example.

**Comment by nclm** [comment]
This could be about adding more categories or always having subcategories? Or doing more with tags?
But in most cases, I feel like most apps have one (or 2?) main “categories” that would be worth linking to. Games or genre of games for games, browsers for browsers, music players for music players, etc. It’s nice to be able to see every other app of the same kind in one click, no matter how many specific tags it also have.

**Comment by razzeee** [comment]
Browsers would be networking, so you would end up not seeing browsers, but apps like https://flathub.org/apps/org.signal.Signal in there other messengers or FTP clients, VPN Clients etc

**Comment by nclm** [comment]
Yes exactly that’s what I meant by more categories or subcategories. In networking, there could be a `Browsers` subcategory of `Networking` in that example. Separate `Music players` from `Video players` in `Audio & Video`, etc.

**Comment by razzeee** [comment]
Adding one subcategory to the xdg-menu spec took 16 month so far. Probably needs artwork and distros to actually start using it, before we can proceed. So please contribute to the spec.

Categories are just not good enough.

**Comment by nclm** [comment]
What about using tags on Flathub and having one tag defined as the main one for each app? Then it would be possible to have a “See all browsers” “See all games” links.

**Comment by razzeee** [comment]
Tags are already a thing, but there is nothing enforced. Picking a main one would also need appstream to allow for that.

Feels too unstructured for me.

**Comment by nclm** [comment]
I guess I have to look up appstream thanks 😄 I’m just a user and not a developper, and that littles piece of feedback on the.navigation of Flathub is getting more technical than expected!

**Comment by razzeee** [comment]
It's mostly unfortunate, that the category spec is sooo old, otherwise this would be a no brainer.

In the end, from the POV of the store, navigation by categories is not ideal. It could be a footnote, but should never be a center piece.
